.nvmrc

```diff
@@ -1 +1 @@
-10.16
+12
```

Comment:
question: why Node 12 and not 14 or 15?

Reply:
I changed it from v10 to v12 to see if any node modules would break. Forgot to update it to v14 :sweat_smile: .

Reply:
Updated to v14!